### DISCUSSION TITLE
Bayesian Elastic Net regression

### DISCUSSION THREAD

**Discussion by Behzad-Iravani** [pymc-devs/pymc3#7260] [Ideas]
Dear Community, 
I'm interested in implementing Bayesian Elastic Net Regression. It seems to me using PyMC resources would be a great starting point. Any suggestions on this matter are greatly appreciated. 
Behzad

**Comment by ricardoV94** [comment]
Hi @Behzad-Iravani, for user questions it's better to use our discourse: https://discourse.pymc.io/

We keep github discussions for development only

PS: Would love to hear more about Bayesian Elastic Nets, so if you can share some resources when you open the discourse thread, that would be great!
